sklearn/metrics/_regression.py

```diff
@@ -342,6 +344,13 @@ def mean_absolute_percentage_error(
     0.5515...
     >>> mean_absolute_percentage_error(y_true, y_pred, multioutput=[0.3, 0.7])
     0.6198...
+    >>> # the value when some element of the y_true is zero
+    >>> # is arbitrarily high because of the division
+    >>> # by epsilon
```

Comment:
Nit: We can rewrap to use more horizontal space:

```suggestion
    >>> # the value when some element of the y_true is zero is arbitrarily high because
    >>> # of the division by epsilon
```